app/views/categories/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<% if notice %>
+  <p class="alert alert-success"><%= notice %></p>
+<% end %>
+<% if alert %>
+  <p class="alert alert-danger"><%= alert %></p>
+<% end %>
+<section class="container">
+  <% if @categories.blank? %>
+    <p>Hello!!! You dont have yet any category on this page please click on the botton below to get one.</p>
+  <% else %>
+    <% @categories.each do |category| %>
+    <%= link_to category_path(category.id) do %>
+    <article class="row d-flex border my-1">
+      <div class="col-3">
+        <img src=<%= category.icon %> alt="Category Icon" class="m-1 img-fluid rounded-1" > 
+      </div>
+      <div class="col-6 pt-3">
+        <p class="category-name"><% category.name %></p>
+        <span class="date" > category.created_at</span>
+      </div>
+    </article>
+  <% end %>
+  <% end %>
+  <% end %>
+</section>
+
+<div class="bottom-container position-fixed bottom-0 w-100">
+  <div class="inner-bottom-container text-center">
+    <%= link_to "NEW CATEGORY", new_category_path, class: "bottom-btn" %>
+  </div>
+</div>
```

Comment:
- [ ] For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.Kindly fix it.
![image](https://user-images.githubusercontent.com/38635158/184506609-b6cad9b9-1b34-4b90-a8b1-c69be9f6bb9e.png)